static/js/components/ShiftManagement.jsx

```diff
@@ -30,6 +30,32 @@ const useStyles = makeStyles(() => ({
   },
 }));
 
+function isDailyShift(shiftName) {
+  const regex = /\d+\/\d+/;
+  return regex.test(shiftName)
```

Comment:
Can we have some data on the shift object (not necessarily in the DB) that tells if it is a daily or not? If you rely on the name alone, you will see some trouble when some user decides to call their shift "not a daily shift 1/1" or something like that.

Reply:
If we dont store it in the DB, i think the name is the only option. What i can add (i should, thought about it already actually), is checking that the name the user inputs doesnt contain anything that matches that regex condition. If it does, the form tells you its an incorrect name. What do you think ? I think that adding another column in the DB to indicate that a shift repeats itself is too much, especially because the only thing we use it for later on is to change the display slightly.

Reply:
When you save a daily shift to the DB, does it get saved as multiple, different shifts, or as a single shift?

Reply:
It creates N shifts in the DB, for the N days you selected. The issue we could have had with creating just one big shift on the whole period + adding a column or some way to say that it is a repeating shift, is that then we could not have different members on each of the daily shifts. We had to share the members for all of the instances/days of a daily shift.

Reply:
I added the regex check !